viz.yaml

```diff
@@ -35,8 +35,8 @@ fetch:
     id: fetch_view_limits
     location: cache/fetch_view_limits.rds
     reader: rds
-    fetcher: fetch_view_limits
-    scripts: [scripts/fetch/fetch_view_limits.R]
+    fetcher: view_limits
+    scripts: [scripts/fetch/view_limits.R]
```

Comment:
👍  I think I can get on board with this. It's OK that we may have a fetcher and a processor of the same name but making sure the location name has fetch on it is important.

Reply:
makes sense. if it were just the location and not also the ID, i'd suggest using cache subfolders like `cache/fetch/view_limits.rds`, but i think the motivation is also for being able to immediately see the phase from a viz item ID , right?

Reply:
I am up for following what the group has decided. I personally prefer unique ids that aren't prefixed, but I also missed the conversation that decided to prefix and don't want to get in the way. 

Also, editorial: `fetch_{something}` isn't the best name for the item you get back from fetching something, {something} is. Or `fetched_` `processed_`, which I don't think is very useful other than being clear about which step the target came from.

Reply:
we're taking (@jread-usgs has taken) this to slack for more discussion